lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_auth_client.rb

```diff
@@ -56,6 +52,12 @@ def get_authorization(google_service_path, firebase_cli_token, debug = false)
 
       private
 
+      def application_default_creds
+        Google::Auth.get_application_default([SCOPE])
```

Comment:
Just to be extra careful, did you confirm that setting `GOOGLE_APPLICATION_CREDENTIALS` to a "service account key file" (JSON) as described in the docs still works? https://firebase.google.com/docs/app-distribution/ios/distribute-fastlane#service-acc-fastlane. I can help with this if you want.

Side note: after this ships we should be able to update the docs to specify that we support ADC in general.

Reply:
Confirmed locally that this still works 👍